eng/pipelines/templates/scripts/verify-path-length.yml

```diff
@@ -22,16 +25,17 @@ steps:
         print('Analyzing length of paths...')
         for root, dirs, files in os.walk('{0}'.format(source_directory)):
             for file in files:
-                file_path = os.path.join(root, file)
-                if (len(file_path) > longest_file_path_length):
-                    longest_file_path_length = len(file_path)
+                file_path = os.path.relpath(os.path.join(root, file), source_directory)
+                if ((len(file_path) + ${{ parameters.BasePathLength }}) > longest_file_path_length):
+                    longest_file_path_length = len(file_path) + ${{ parameters.BasePathLength }}
                     longest_file_path = file_path
                     if (longest_file_path_length >= 260):
-                        raise Exception('{0} : is 260 or more Characters long. Reduce path length'.format(longest_file_path))
-                if (len(root) > longest_dir_path_length):
-                    longest_dir_path_length = len(root)
-                    longest_dir_path = root
+                        raise Exception('{0} : is 210 or more Characters long. Reduce path length'.format(longest_file_path))
```

Comment:
Need to update this error message.

Reply:
Its a little confusing what to make it, am adding the length of the basepath to determine the full path length. But to the user only the relative path from the repo really matters. So they only really need to care about having that below 210 since they are not checking their basepath into the repo.

Reply:
I would say something like "With a base path length of {} your path {} is longer then 260 chars long. Reduce path length".